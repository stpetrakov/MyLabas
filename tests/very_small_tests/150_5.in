32497 23381 19282 15117 32568 16727 13576 14842 8322 3206 10903 10042 30013 9765 20101 16458 5567 4803 27257 25157 31883 28011 19871 32437 30672 8988 27214 21160 21555 632 30441 5127 9422 29632 5635 2340 19577 27904 18886 16018 11336 30343 13973 20281 6997 21402 30960 29353 3007 9066 4760 17839 242 3651 13044 294 15145 21253 24244 4880 15410 18648 15028 19942 2417 10524 32291 8748 20264 13727 20452 5703 18678 3859 4916 28819 5021 20415 26510 18353 19464 15988 24523 27934 8076 11715 13454 26122 12654 23405 30175 26215 31141 25627 22 576 226 8015 14331 2380 7630 30619 21035 16579 6034 4881 10686 28528 20893 24570 27174 28680 25761 22987 4186 15802 18655 6740 1845 12039 30117 2875 25368 21204 6677 19369 215 9149 21287 7747 3027 1453 754 29532 12131 16811 11559 25448 30585 29687 31966 11700 9346 8278 22421 3925 30349 24746 8565 1631 