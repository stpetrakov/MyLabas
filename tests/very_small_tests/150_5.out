22 215 226 242 294 576 632 754 1453 1631 1845 2340 2380 2417 2875 3007 3027 3206 3651 3859 3925 4186 4760 4803 4880 4881 4916 5021 5127 5567 5635 5703 6034 6677 6740 6997 7630 7747 8015 8076 8278 8322 8565 8748 8988 9066 9149 9346 9422 9765 10042 10524 10686 10903 11336 11559 11700 11715 12039 12131 12654 13044 13454 13576 13727 13973 14331 14842 15028 15117 15145 15410 15802 15988 16018 16458 16579 16727 16811 17839 18353 18648 18655 18678 18886 19282 19369 19464 19577 19871 19942 20101 20264 20281 20415 20452 20893 21035 21160 21204 21253 21287 21402 21555 22421 22987 23381 23405 24244 24523 24570 24746 25157 25368 25448 25627 25761 26122 26215 26510 27174 27214 27257 27904 27934 28011 28528 28680 28819 29353 29532 29632 29687 30013 30117 30175 30343 30349 30441 30585 30619 30672 30960 31141 31883 31966 32291 32437 32497 32568 